### ISSUE TITLE
why the weight decay is equal to 3.0517578125e-05 (1/32768).

### ISSUE THREAD

**Issue by chichan01** [NVIDIA/DeepLearningExamples#84]
Hi,
   Would you like to tell me why you set the weight decay is 1/32768

**Comment by srutownik** [comment]
Please, specify, which model you are referring to.

**Comment by chichan01** [comment]
Hi,
  Thanks for your response... all scripts under 
DeepLearningExamples/PyTorch/Classification/RN50v1.5/resnet50v1.5/training set wd to 1/32768
and the model is resnet50v1.5

**Comment by mwawrzos** [comment]
In Depp Learning Examples we use the optimizations of the training process if our experiments confirmed that its advantage is notable, stable and does not reduce generalization capability.

The value of weight decay used in the ResNet scripts was chosen from various values tested for the scenario described in the [README](https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/Classification/RN50v1.5/README.md). The value 1/32768 resulted in the best trade-off between the training time and achieved accuracy.

For different training scenarios (different training dataset, different training hyper-parameters) there might exist more optimal values of hyper-parameters, like weight decay. However, the values used in the ResNet Deep Learning Example definitely should be satisfying, and it will be a good start point for possible hyper-parameters fine-tuning.

Is this answer satisfying for you?
